
cam_num: 1
img_topic_0: /camera/rgb

# 070305
Tcl_0: [
  -0.00147, -0.99997, -0.00743, 0.02627,
  0.00889, 0.00742, -0.99993, -0.00389,
  0.99996, -0.00154, 0.00887, -0.00384,
  0.00000, 0.00000, 0.00000, 1.00000
]

cam_0: 
  cam_model: FishPoly
  image_width: 1600
  image_height: 1296
  k2: -1.8624099983801271e-02
  k3: 6.8388604097113198e-02
  k4: -1.8834920443612091e-01
  k5: 2.2924951610547040e-01
  k6: -1.4642373783883955e-01
  k7: 3.6521540148504360e-02
  p1: 0.
  p2: 0.
  A11: 7.3578621748998717e+02
  A12: 5.0313730284836411e-01
  A22: 7.3622293197097224e+02
  u0: 7.8605471995455093e+02
  v0: 6.5758820486030027e+02
  isFast: 0
  numDiff: 3000`1
  maxIncidentAngle: 120
